service/src/test/java/uk/gov/hmcts/reform/fpl/service/sdo/StandardDirectionsOrderServiceTest.java

```diff
@@ -98,15 +99,16 @@ void shouldUseReplacementSDOForTemporaryStandardDirectionOrder() {
 
     @Test
     void shouldUseCurrentSDODocumentForTemporaryStandardDirectionOrder() {
+        StandardDirectionOrder previousSDO = StandardDirectionOrder.builder()
+            .orderDoc(SEALED_DOC)
```

Comment:
it is a bit misleading, at this point sdo should not be sealed yet, maybe just WORD_DOC